concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/00 A Drop-Down Editor Does Not Show Data.md

```diff
@@ -0,0 +1,9 @@
+This section describes the issues related to the following drop-down editors:
+
+- [SelectBox](/Documentation/Guide/UI_Components/SelectBox/Getting_Started_with_SelectBox/)
+- [Lookup](/Documentation/Guide/UI_Components/Lookup/Getting_Started_with_Lookup/)
+- [DropDownBox](/Documentation/Guide/UI_Components/DropDownBox/Getting_Started_with_DropDownBox/)
+- [TagBox](/Documentation/Guide/UI_Components/TagBox/Getting_Started_with_TagBox/)
+- [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/)/[TreeList](/Documentation/Guide/UI_Components/TreeList/Getting_Started_with_TreeList/) lookup column editor
+
+Drop-down editors may not display data in different scenarios: on save, on first load, when the value property is changed in the code. They may also display different values, for example, IDs instead of text. Follow the steps below to learn how to troubleshoot your application. If the solutions listed here do not help, create a ticket in our Support Center and describe your issue in details: [Create a ticket](https://supportcenter.devexpress.com/ticket/create).
```

Comment:
I'm confused about a couple of things. 

- "may not display data" - does this mean "display an empty edit box" or "display an empty dropdown list", or both, or maybe even something else? What I'm saying is "no data" is a big vague for a drop-down editor. Such an editor displays data in multiple places. We can be more specific about the issue. 
- "on save". I'm not sure what you imply. An editor displays data. Then I save some data changes (do they have to be changes in this editor or not?) and then the data disappears? Only during the save operation? Or data disappears forever? .... One can ask similar question about "when the value property is changed" or "on first load". I mean - does the data appear on subsequent loads? :-) In any event - I don't really see you reference those "scenarios" in the next sections. Maybe there's no reason to mention them here at all. Since the troubleshooting guide doesn't ask me to identify such a "scenario" - then why are we listing them here?